src/components/client/support-form/SupportForm.tsx

```diff
@@ -222,21 +222,22 @@ export default function SupportForm() {
       onSubmit={handleSubmit}
       initialValues={initialValues}
       validationSchema={validationSchema[activeStep]}
-      innerRef={formRef}>
-      <Hidden mdDown>
-        <Stepper
-          alternativeLabel
-          activeStep={activeStep}
-          connector={<StepConnector sx={{ mt: 1.5 }} />}>
-          {steps.map((step, index) => (
-            <Step key={index}>
-              <StepLabel error={isStepFailed(index)} StepIconComponent={StepIcon}>
-                {t(step.label)}
-              </StepLabel>
-            </Step>
-          ))}
-        </Stepper>
-      </Hidden>
+      innerRef={formRef}
+      validateOnBlur={false}
+      validateOnChange={currentValidatedField === activeStep}>
```

Comment:
`currentValidateField` sounds like it pertains to a field, but in here it is used to toggle on/off of individual field validation on a step. 

How about `validateInidividualFieldsOnStep` ?

Reply:
I'd suggest to make the pattern more explicit and readable by making the value that comes out of the ValidationObserver a map such that individual field validation is turned on/off explicitly 
```tsx 
<GeneralForm<...>
      ...
      validateOnBlur={validateIndividualFields[activeStep]}
      validateOnChange={validateIndividualFields[activeStep]}>
      
```